src/gwt/src/org/rstudio/studio/client/workbench/prefs/views/python/PythonInterpreterListEntryUi.ui.xml

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE ui:UiBinder SYSTEM "http://dl.google.com/gwt/DTD/xhtml.ent">
+<ui:UiBinder xmlns:ui="urn:ui:com.google.gwt.uibinder"
+	xmlns:g="urn:import:com.google.gwt.user.client.ui">
+	
```

Comment:
Noticed this file is indented with tabs instead of spaces.

Reply:
*shakes fist at Eclipse*